tests/old/account/test_widgets.py

```diff
@@ -51,6 +53,7 @@ def test_widget_appearance(self):
         response = self.client.get('/dashboard/')
         self.assertContains(response, 'My personal info', status_code=200)
 
+    @skip
```

Comment:
You could have skipped a hole test-class by putting the decorator in front of the class. But your way works, too.